`@html-eslint/require-closing-tags` incompatible with Prettier

Additionally to #106 (still an issue in 0.54.0 following ea754950c4bc130aab5531e93539736b628af964):

```js
// eslint.config.mjs
import xo from "eslint-config-xo";

export default xo({ prettier: "compat" });
```

```console
$ >test.html echo '<br>'
$ prettier -w test.html
test.html 17ms
$ cat test.html
<br />
$ eslint test.html

/tmp/test/test.html
  1:5  error  Unexpected self closing tag for br  @html-eslint/require-closing-tags

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
$ eslint --fix test.html
$ cat test.html
<br >
$ prettier -w test.html
test.html 15ms
$ cat test.html
<br />
```

--- a/source/html.js
+++ b/source/html.js
@@ -14,6 +14,7 @@ const prettierConflictingRules = [
 	'@html-eslint/no-multiple-empty-lines',
 	'@html-eslint/no-trailing-spaces',
 	'@html-eslint/quotes',
+	'@html-eslint/require-closing-tags',
 ];
 
 export function getHtmlConfig({space = false, prettier = false} = {}) {

